terraso_backend/apps/shared_data/visualization_tileset_tasks.py

```diff
@@ -0,0 +1,161 @@
+import csv
+import json
+import threading
+
+import openpyxl
+import structlog
+from django.conf import settings
+
+from apps.core.gis.mapbox import create_tileset, remove_tileset
+from apps.shared_data.services import data_entry_upload_service
+
+from .models import VisualizationConfig
+
+logger = structlog.get_logger(__name__)
+
+
+class AsyncTaskHandler:
+    def start_task(self, method, args):
+        t = threading.Thread(target=method, args=[*args], daemon=True)
+        t.start()
+
+
+def start_create_mapbox_tileset_task(visualization_id):
+    AsyncTaskHandler().start_task(create_mapbox_tileset, [visualization_id])
+
+
+def start_remove_mapbox_tileset_task(tileset_id):
+    AsyncTaskHandler().start_task(remove_mapbox_tileset, [tileset_id])
+
+
+def get_rows_from_file(data_entry):
+    type = data_entry.resource_type
+    if type.startswith("csv"):
+        file = data_entry_upload_service.get_file(data_entry.s3_object_name, "rt")
+        reader = csv.reader(file)
+        return [row for row in reader]
+    elif type.startswith("xls"):
+        file = data_entry_upload_service.get_file(data_entry.s3_object_name, "rb")
+        wb = openpyxl.load_workbook(file)
```

Comment:
question: Would there be any advantage to using the pandas library here? I'm pretty sure we already have it as a dependency, and it seems like it is able to load [excel files](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html). But maybe the use case here is simple enough that it's overkill.

Reply:
I didn't know there was a library already in the code that we could use to read excel files, I will check it

Reply:
I changed the code to use Pandas